File: AccountantWeb/Models/Category.cs
using System.ComponentModel.DataAnnotations;
using System;

namespace Accountant.Models
{
    public class Category
    {
        public int ID { get; set; }

        [Required]
        public string Name { get; set; }
        public string Description { get; set; }

        public DateTime lastModified { get; set; }
    }
}

File: AccountantAndroid/app/src/main/java/com/kunkliricsi/accountant/local/entities/Category.java
package com.kunkliricsi.accountant.local.entities;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "categories")
public class Category {

    @PrimaryKey
    public int id;

    @ColumnInfo(name = "name")
    public String Name;

    @ColumnInfo(name = "description")
    public String Description;
}
